config/nunjucks/globals.js

```diff
@@ -23,6 +23,10 @@ const footerItems = [
     href: ENABLE_COMPONENTS_LIBRARY ? componentsUrl : undefined,
     text: i18n.t('components::title'),
   },
+  {
+    href: '/whats-new',
+    text: "What's new",
+  },
```

Comment:
We might want to discuss whether it's the right time to launch this link, when we don't yet have the content written for this page - I suspect we might want to launch it later.

Reply:
Makes sense, Ill remove it for now